dynomitemanager/src/main/java/com/netflix/dynomitemanager/sidecore/utils/RetryableCallable.java

```diff
@@ -37,6 +37,8 @@ public RetryableCallable()
 
     public RetryableCallable(int retrys, long waitTime)
     {
+    	if(retrys<=-1) throw new RuntimeException(new IllegalAccessException("Retrys should be >= 0"));
```

Comment:
Do we have a case that retrys will be <-1? What is the reason for this safety check?